pkg/renderer/sgml/html5/templates.go

```diff
@@ -0,0 +1,84 @@
+package html5
+
+import (
+	"github.com/bytesparadise/libasciidoc/pkg/renderer/sgml"
+	"io"
+)
+
+var templates = &sgml.Templates{
```

Comment:
cool, that's the kind of structure I was also thinking about. And it's even better here since it only contains the "string" templates (ie, not initialized at this point, that's something that can be done in the `sgml` package for all backends, indeed)